readme.md

```diff
@@ -62,6 +62,16 @@ This is used for deployment or deployment package testing.
    + Windows: `mvnw.cmd package`
 3. Find the output in the root of the `target` folder
 
+### Running in Offline Mode
```

Comment:
The download of the offline static content would need to be done prior to the demo (assuming it's offline) right?